src/Sylius/Bundle/AdminBundle/Resources/config/app/twig_hooks/shipping_category/create.yaml

```
@@ -0,0 +1,23 @@
+twig_hooks:
+    hooks:
+        'sylius_admin.shipping_category.create.content':
+            form:
+                template: '@SyliusAdmin/shared/crud/common/content/form.html.twig'
+                configuration:
+                    render_rest: false
+
+        'sylius_admin.shipping_category.create.content.form':
+            sections:
+                template: '@SyliusAdmin/shipping_category/form/sections.html.twig'
+
+        'sylius_admin.shipping_category.create.content.form.sections':
+            details:
+                template: '@SyliusAdmin/shipping_category/form/sections/details.html.twig'
```

To be consistent with our unification (ref.: https://github.com/Sylius/Sylius/pull/16568), it should be:
```suggestion
            general:
                template: '@SyliusAdmin/shipping_category/form/sections/general.html.twig'
```
and change the corresponding directory and paths